src/AssembleNGPNodeSolverAlgorithm.C

```diff
@@ -91,10 +91,8 @@ AssembleNGPNodeSolverAlgorithm::execute()
   const stk::mesh::EntityRank entityRank = stk::topology::NODE_RANK;
   const int rhsSize = rhsSize_;
 
-#ifdef KOKKOS_ENABLE_CUDA
   CoeffApplier* coeffApplier = eqSystem_->linsys_->get_coeff_applier();
   CoeffApplier* deviceCoeffApplier = coeffApplier->device_pointer();
```

Comment:
Are you thinking that `CoeffApplier` returned currently by `get_coeff_applier()` will be managed internally within LinearSystem and will in future return the pointer currently accessed by `device_pointer()`? That would be a good cleanup.

Reply:
Yes I agree that would be a good next step.